### ISSUE TITLE
Comments on Android Fiddler

### ISSUE THREAD

**Issue by mu88** [mu88/mu88.github.io#6]

**Comment by Vishrut-Arora** [comment]
To be very precise it is a web debugging proxy tool. Rest all depends on your requirement. Do you want to do something specific?

**Comment by mu88** [comment]
Yes, I was analyzing the traffic of the Bose SoundTouch app to reverse-engineer which (undocumented) WebSocket commands are transferred for a certain user interaction.
